tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -721,16 +720,13 @@ <h2>Continue...</h2>
         this.$.continueDialog.open();
       },
 
-      _showSourceCode() {
+      _showSourceCodeChanged(event) {
+        this.set('_sourceCodeShown', event.target.active);
         if (this._sourceCodeShown) {
-          this.$$('#node-entries').style.height = '80%';
-          this.set('_sourceCodeShown', false);
-          this.set('_toggleSourceCodeButtonText', 'Show Code >>');
-        } else {
           this.$$('#node-entries').style.height = '40%';
-          this.set('_sourceCodeShown', true);
-          this.set('_toggleSourceCodeButtonText', 'Hide Code <<');
           this.$.sourceCodeView.render();
+        } else {
+          this.$$('#node-entries').style.height = '80%';
```

Comment:
It's odd how `Polymer.dom(this.$$('#node-entries')).classList.add('expanded-node-entries')` does not work ...

Reply:
Ack. We have a tracking bug to look into that later.